src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -884,6 +886,13 @@ private Future<Void> verifyDeleteAllowed(PoLine line, RequestContext requestCont
         .compose(order -> protectionService.isOperationRestricted(order.getAcqUnitIds(), DELETE, requestContext)));
   }
 
-
+  private Future<Void> createShadowInstanceIfNeeded(CompositePoLine compositePoLine, RequestContext requestContext) {
```

Comment:
Ensure that extra costs are not incurred for non-ECS flow. We should check if ECS is enabled before attempting to create shadow instances. This should exclude needing to make a call to mod-inventory to see if the shadow instance exists.

Reply:
Done